packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -298,17 +306,27 @@ function FontLibraryProvider( { children } ) {
 				refreshLibrary();
 			}
 
-			if ( installationErrors.length > 0 ) {
+			if ( installationErrors.length === 1 ) {
 				throw new Error(
 					sprintf(
 						/* translators: %s: Specific error message returned from server. */
+						__( 'There was an error installing fonts. %s' ),
+						installationErrors[ 0 ]
+					)
+				);
+			} else if ( installationErrors.length > 1 ) {
+				throw new Error(
+					sprintf(
+						/* translators: %s: Specific error messages returned from server. */
 						__( 'There were some errors installing fonts. %s' ),
-						installationErrors.reduce(
-							( errorMessageCollection, error ) => {
-								return `${ errorMessageCollection } ${ error.message }`;
-							},
-							''
-						)
+						'<ul>' +
```

Comment:
Isn't it a bit weird to use HTML in error messages?

Reply:
Simpler:

```
async function installFonts() { throw { errors: installationErrors } };
installFonts().catch( ( ( { errors } ) => console.log( errors ) );
```

Reply:
I should have checked this more carefully and you are of course correct.

We should format the errors are the point of _rendering_ and not in the place where they are processed.

Ideally the component which renders the errors should receive the raw data and then choose how it renders rather than printing HTML.

Reply:
@pbking I moved the error formatting in https://github.com/WordPress/gutenberg/pull/59564/commits/18c1af78e782ce5ba87c814cd2ea39f4c50bf84e.

So the errors object that is returned will contain the same error message and potentially the dedicated messages about the specific errors.

I did consider returning [`AggregateError`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError) but that looks to be too new to rely on at this time.

Please feel free to tidy and improve my code or reject it.